components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.commons/src/main/java/org/wso2/carbon/cluster/coordinator/commons/configs/CoordinationStrategyConfiguration.java

```diff
@@ -58,6 +58,17 @@ public String getConfiguration() {
         return null;
     }
 
+    /**
+     * Method to retrieve DataSource configurations.
```

Comment:
Need newline after method description.

Reply:
This code segment is obsolete with later changes.